app/src/main/java/com/example/habitsmasher/ui/dashboard/AddHabitDialog.java

```diff
@@ -45,6 +45,16 @@
     private EditText _habitReasonEditText;
     private TextView _habitDateTextView;
 
+    private Button _mondayButton;
+    private Button _tuesdayButton;
+    private Button _wednesdayButton;
+    private Button _thursdayButton;
+    private Button _fridayButton;
+    private Button _saturdayButton;
+    private Button _sundayButton;
```

Comment:
Privatized all of the buttons so that I can extract the setting of them to their own method.